test/functionalTest/cases/1316_create_subscription/post_subscriptions.test

```diff
@@ -228,8 +228,8 @@ Date: REGEX(.*)
     "id": "REGEX([0-9a-f]{24})",
     "notification": {
         "attrs": [
-            "humidity",
-            "temperature"
+            "temperature",
```

Comment:
Before the fix, the attributes always came in alphabetic order. This made ordering for **values** impossible and was fixed. 

About pretty-print ... I thought the order in vectors weren't touched by the pretty-print ...